tests/query/query-ctor_error-001.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+MongoDB\Driver\ReadConcern construction (invalid readConcern type)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
```

Comment:
No need. This exception is something we check for and throw ourselves in the ctor implementation.